Formula/libquicktime.rb

```diff
@@ -28,6 +28,14 @@ class Libquicktime < Formula
   end
   patch :DATA
 
+  # Fix CVE-2016-2399. Applied upstream on March 6th 2017.
```

Comment:
Us Brits tend to use this format in more formal settings as well 😸. The biggest difference I'm aware of between our two dates is the shortest one, 31/10/17 Vs 10/31/17, etc.